assets/src/components/RangeSlider.js

```diff
@@ -35,15 +35,17 @@ const rangeSlider = props => {
         range
         activeDotStyle={activeDotStyle}
         dotStyle={dotStyle}
-        handleStyle={[activeDotStyle, activeDotStyle]}
-        railStyle={unselectedStyle}
-        trackStyle={[selectedStyle]}
```

Comment:
These are deprecated in the new version of rc-slider and includes as part of the styles property.